data/transactions/logic/opcodes.go

```diff
@@ -596,6 +606,9 @@ var OpSpecs = []OpSpec{
 	// randomness support
 	{0xd0, "vrf_verify", opVrfVerify, proto("bbb:bi"), randomnessVersion, field("s", &VrfStandards).costs(5700)},
 	{0xd1, "block", opBlock, proto("i:a"), randomnessVersion, field("f", &BlockFields)},
+
+	// switch on value
+	{0xe0, "switchi", opSwitchInt, proto("i:"), 8, opSwitch()},
```

Comment:
Nothing to change at the moment, but I'm thinking that if we do the other kind of switch too, good names for the opcodes would be `switch_dense` (this one) and `switch_sparse` (the one with values and labels intermixed). But would could provide a pseudo-op `switch` that figured out which one was being assembled by looking at its arguments.